config/git2consul.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "1.0",
+    "repos" : [{
+        "name" : "config/store",
+        "url" : "https://github.com/ahmadiq/storeconfig.git",
```

Comment:
yes we can put configs in one repo and use different paths for each microservice.

For using separate config files for each moicroservice, might need some code modifications to git2consul. i'll check, as currently i think they only support using one file.